lib/src/model.dart

```diff
@@ -4548,3 +4864,313 @@ class TypeParameter extends ModelElement {
   @override
   String toString() => element.name;
 }
+
+/// Everything you need to instantiate a Package object for documenting.
+class PackageBuilder {
```

Comment:
+1, this, class `Canonicalization`, class `Privacy` - I think the modeling improvements here will really help.